src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.figures;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        Figure figure;
+
+        int figureInt = random.nextInt(5);
+        switch (figureInt) {
+            case 0:
+                figure = new Circle(random.nextInt(30), colorSupplier.getRandomColor());
+                break;
```

Comment:
```suggestion
                return new Circle(random.nextInt(SIZE_LIMIT), colorSupplier.getRandomColor());
```
Please do the same for other cases. Move "30" to constant